package.json

```diff
@@ -34,7 +34,7 @@
   "devDependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.2.0",
     "@edx/browserslist-config": "1.2.0",
-    "@edx/frontend-build": "12.8.27",
+    "@edx/frontend-build": "github:openedx/frontend-build#ags/2321",
```

Comment:
https://github.com/openedx/frontend-build/pull/365

Reply:
This will presumably change prior to merging?

Reply:
@davidjoy Yes. the dependent `@edx/frontend-build` PR on associated with that `ags/2321` branch will need to be merged in prior to this PR merging.